homeassistant/components/mqtt/strings.json

```diff
@@ -181,7 +181,7 @@
         },
         "qos": {
           "name": "QoS",
-          "description": "Quality of Service to use. O. At most once. 1: At least once. 2: Exactly once."
+          "description": "Quality of Service to use. 0. At most once. 1: At least once. 2: Exactly once."
```

Comment:
Just maybe nitpicking here, but then we can also change the dot after the `0` to `:` 🙈

Reply:
Correct, I not noticed it as focused on number instead of letter. I will refix it again :)